From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Fri, 17 Mar 2023 01:52:14 +0000
Subject: [PATCH] remove dummy test files


diff --git a/app/src/androidTest/java/com/akansh/fileserversuit/ExampleInstrumentedTest.java b/app/src/androidTest/java/com/akansh/fileserversuit/ExampleInstrumentedTest.java
deleted file mode 100644
index d451305..0000000
--- a/app/src/androidTest/java/com/akansh/fileserversuit/ExampleInstrumentedTest.java
+++ /dev/null
@@ -1,27 +0,0 @@
-package com.akansh.fileserversuit;
-
-import android.content.Context;
-
-import androidx.test.platform.app.InstrumentationRegistry;
-import androidx.test.ext.junit.runners.AndroidJUnit4;
-
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import static org.junit.Assert.*;
-
-/**
- * Instrumented test, which will execute on an Android device.
- *
- * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
- */
-@RunWith(AndroidJUnit4.class)
-public class ExampleInstrumentedTest {
-    @Test
-    public void useAppContext() {
-        // Context of the app under test.
-        Context appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
-
-        assertEquals("com.akansh.fileserversuit", appContext.getPackageName());
-    }
-}
diff --git a/app/src/test/java/com/akansh/fileserversuit/ExampleUnitTest.java b/app/src/test/java/com/akansh/fileserversuit/ExampleUnitTest.java
deleted file mode 100644
index dc9868e..0000000
--- a/app/src/test/java/com/akansh/fileserversuit/ExampleUnitTest.java
+++ /dev/null
@@ -1,17 +0,0 @@
-package com.akansh.fileserversuit;
-
-import org.junit.Test;
-
-import static org.junit.Assert.*;
-
-/**
- * Example local unit test, which will execute on the development machine (host).
- *
- * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
- */
-public class ExampleUnitTest {
-    @Test
-    public void addition_isCorrect() {
-        assertEquals(4, 2 + 2);
-    }
-}
\ No newline at end of file
-- 
2.30.2

